code/_onclick/item_attack.dm

```diff
@@ -31,19 +31,26 @@ avoid code duplication. This includes items that may sometimes act as a standard
 
 // No comment
 /atom/proc/attackby(obj/item/W, mob/user, var/click_params)
+	user.client.interact_log += "([time_stamp()]) - Interacted with [src], using [W]. (Intent: [user.a_intent]|Location: [get_area(src)])"
 	return
 
 /atom/movable/attackby(obj/item/W, mob/user)
 	if(W?.item_flags & ITEM_FLAG_NO_BLUDGEON)
 		visible_message("<span class='danger'>[src] has been hit by [user] with [W].</span>")
-		return
```

Comment:
Random line remove.

Reply:
No, not random on second look, I removed it because I was going to return ..() there but the next line is a return ..() anyways, I will comment